### PR TITLE
APIS-896 Addition of Stub run mode to GateKeeperFrontend

### PR DESCRIPTION
**Pull request by AndyBowes-EqualExperts**
Addition of multiple run.modes to allow the GateKeeperFrontend to be started in "Stub" which uses the API-Services-Stub to supply the Third Party Application & Developer resources.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -30,17 +30,6 @@ json.encryption {
   previousKeys = []
 }
 
-auditing {
```
Comment (rm-hull) [inline conf/application.conf]: would be inclined to leave the audit alone, and override it in stub mode only

File: conf/application.conf [hunk 2]
```diff
@@ -58,27 +47,147 @@ contact-frontend {
...
+Stub {
+  auditing {
+    enabled=false
+  }
+
+  microservice {
+    services {
+      auth {
+        host = localhost
+        port = 8500
```
Comment (rm-hull) [inline conf/application.conf]: should the auth service port be 11111 as well in stub mode?

File: conf/application.conf (hunk 1 above)
Comment (rm-hull) [inline conf/application.conf]: actually, ignore that, i think its more explicit/clearer like you've done it

Comment (rm-hull) [inline conf/application.conf]: and just seen your slack comments :)

File: conf/application.conf (hunk 2 above)
Comment (AndyBowes-EqualExperts) [inline conf/application.conf]: The Auth service has not been implemented in the API Service Stub so we are using the standard 'sm --start AUTH' which will serve on port 8500.

We could look at extending the API Service Stub to implement the necessary entry points as a separate JIRA.

**Pull request merged**